torch/fx/experimental/proxy_tensor.py

```diff
@@ -657,8 +657,11 @@ def disable_autocast_cache():
         torch.set_autocast_cache_enabled(old_value)
 
 
-def make_fx(f, decomposition_table=None, tracing_mode="real", _allow_non_fake_inputs=False):
-    assert tracing_mode in ["real", "fake", "symbolic"]
+def make_fx(f, decomposition_table=None, tracing_mode="real", _allow_non_fake_inputs=False, existing_fake_mode=None):
+    assert tracing_mode in ["real", "fake", "symbolic", "existing"]
```

Comment:
`fake` and `existing` looks duplicated. When caller specifying `existing`, I assume the code will create FakeTensorMode if it doesn't exist. Since there is no document, I don't quite understand the design concepts and expected behaviors of those options.